docs/api-inference/tasks/text-classification.md

```diff
@@ -38,63 +38,66 @@ Explore all available models and find the one that suits you best [here](https:/
 
 <curl>
 ```bash
-curl https://api-inference.huggingface.co/models/distilbert/distilbert-base-uncased-finetuned-sst-2-english \
-	-X POST \
-	-d '{"inputs": "I like you. I love you"}' \
-	-H 'Content-Type: application/json' \
-	-H 'Authorization: Bearer hf_***'
+curl https://api-inference.huggingface.co/models/mixedbread-ai/mxbai-rerank-base-v1 \
+    -X POST \
+    -d '{"inputs": [{"text": "What is Paris?", "text_pair": "Paris is the capital of France"}]}' \
+    -H 'Content-Type: application/json' \
+    -H 'Authorization: Bearer hf_***'
 ```
 </curl>
 
 <python>
-```py
+```python
 import requests
 
-API_URL = "https://api-inference.huggingface.co/models/distilbert/distilbert-base-uncased-finetuned-sst-2-english"
+API_URL = "https://api-inference.huggingface.co/models/mixedbread-ai/mxbai-rerank-base-v1"
 headers = {"Authorization": "Bearer hf_***"}
 
 def query(payload):
-	response = requests.post(API_URL, headers=headers, json=payload)
-	return response.json()
-	
+    response = requests.post(API_URL, headers=headers, json=payload)
+    return response.json()
+    
 output = query({
-	"inputs": "I like you. I love you",
+    "inputs": [
+        {"text": "What is Paris?", "text_pair": "Paris is the capital of France"},
+        {"text": "What is Paris?", "text_pair": "London is the capital of England"}
+    ]
 })
 ```
-
-To use the Python client, see `huggingface_hub`'s [package reference](https://huggingface.co/docs/huggingface_hub/package_reference/inference_client#huggingface_hub.InferenceClient.text_classification).
```

Comment:
Could we use a more relevant package ref, like: https://huggingface.co/docs/huggingface_hub/package_reference/inference_client#huggingface_hub.InferenceClient.sentence_similarity

instead of removing it.

Reply:
Sentence Similarity works a bit differently behind the scenes sadly, I believe. Text Classification (of pairs) is actually what's happening behind the scenes.

Reply:
Perfect! Thanks for the clarification. In that case, let's keep it.

Reply:
Hey @Muhtasham - can you update the PR to keep the ref to HFH lib?